### ISSUE TITLE
Help with error: Mpociot\BotMan\Drivers\hash_equals()

### ISSUE THREAD

**Issue by daltonsutton** [mpociot/botman#243]
I'm trying to install BotMan, and I did so using composer. But I keep getting this error:

`
Fatal error: Call to undefined function Mpociot\BotMan\Drivers\hash_equals() in /Applications/MAMP/htdocs/chat.dev/vendor/mpociot/botman/src/Mpociot/BotMan/Drivers/FacebookDriver.php on line 83`

Can someone point out what I'm doing wrong and how I should go about fixing this? 

I have the following:

```
<?php

require 'vendor/autoload.php';

use Mpociot\BotMan\BotManFactory;
use Mpociot\BotMan\BotMan;

$botconfig = [
    'facebook_token' => 'token',
    'facebook_app_secret' => 'secret',
];

// create an instance
$botman = BotManFactory::create($botconfig);

$botman->verifyServices('token');
```

I've generated my Facebook app and tokens and everything, but facebook won't let me use my ngrok.io domain name because of the fatal error I'm getting.

Any help would be much appreciated!

**Comment by mpociot** [comment]
Hi,

You need at least PHP 5.6 in order to use BotMan. But it seems like the minimum version is wrong in the composer.json file. 
I will update the composer file to state that you need 5.6. 

So if you want to fix this issue, you need at least PHP 5.6

**Comment by daltonsutton** [comment]
Thank you so much! Such a simple thing that I overlooked last night!
